presto-main/src/main/java/com/facebook/presto/server/ResourceGroupStateInfoResource.java

```diff
@@ -68,6 +68,27 @@ public ResourceGroupInfo getQueryStateInfos(@PathParam("resourceGroupId") String
         throw new WebApplicationException(NOT_FOUND);
     }
 
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Encoded
+    @Path("getResourceGroupInfo/{resourceGroupId: .+}")
+    public ResourceGroupInfo getStaticResourceGroupInfo(@PathParam("resourceGroupId") String resourceGroupIdString)
```

Comment:
question: Why do we need a separate method? It is technically very similar to the `getQueryStateInfos`. It feels like it is possible to simply add a parameter to the `getQueryStateInfos`. 

For example: 
  - `?includeQueryInfo=true|false`
  - `?summarizeSubgroups=true|false`
 - `?includeDynamicSubgroups=true|false`

Or maybe something like:

  - `?subgroupsInfo={SUMMARIZE|STATIC|ALL}`